test/plugin/constants.rb

```diff
@@ -106,6 +106,30 @@ module Constants
   INSERT_ID = 'fah7yr7iw64tg857y'.freeze
   INSERT_ID2 = 'fah7yr7iw64tgaeuf'.freeze
 
+  STACKDRIVER_TRACE_ID = '1234567890abcdef1234567890abcdef'.freeze
+  FULL_STACKDRIVER_TRACE = \
+    "projects/#{PROJECT_ID}/traces/#{STACKDRIVER_TRACE_ID}".freeze
+
+  # Invalid trace id for stackdriver.
+  EMPTY_STRING = ''.freeze
```

Comment:
Personally, I think this one is overkill, but I'll let @qingling128 make this call.

Reply:
Haha, i'm fine with defining an empty string constant. It could be shared by others as well though, not necessarily `Invalid trace id for stackdriver`.